packages/hooks/use-attrs/index.ts

```diff
@@ -9,29 +11,26 @@ interface Params {
 const DEFAULT_EXCLUDE_KEYS = ['class', 'style']
 const LISTENER_PREFIX = /^on[A-Z]/
 
-export default (params: Params = {}) => {
+export default (params: Params = {}): ComputedRef<Record<string, unknown>> => {
   const { excludeListeners = false, excludeKeys = [] } = params
-  const instance = getCurrentInstance()
-  const attrs = shallowRef({})
   const allExcludeKeys = excludeKeys.concat(DEFAULT_EXCLUDE_KEYS)
 
-  // Since attrs are not reactive, make it reactive instead of doing in `onUpdated` hook for better performance
-  instance.attrs = reactive(instance.attrs)
```

Comment:
The original method will change internal variable, maybe breaking in the future.

Reply:
Just confirmed. 👍